### PR TITLE
[Rec-IM] more status options

### PR DESCRIPTION
**Pull request by antoniavonto**
Implemented the change status  drop down button in the admin view of participants
image

### CODE REVIEW

**Comment by bennettforkner** [comment]
Hello @antoniavonto -- if you are proposing changes to the recim branch, please set that as the "base" for your pull request instead of develop. It should be a red flag that your PR has 230 commits that are to be added.

**Comment by amos-cha** [comment]
> Hello @antoniavonto -- if you are proposing changes to the recim branch, please set that as the "base" for your pull request instead of develop. It should be a red flag that your PR has 230 commits that are to be added.

👍🏼

File: .env.development [hunk 1]
```diff
@@ -4,8 +4,11 @@
...
 # @TRAIN
- VITE_API_URL=https://360ApiTrain.gordon.edu/
+# VITE_API_URL=https://360ApiTrain.gordon.edu/
 
 # @LOCALHOST
-# VITE_API_URL=http://localhost:51626/
+ VITE_API_URL=http://localhost:51626/
+
+# @Rec-IM
+#VITE_API_URL=https://360apirecim.gordon.edu/
```
Comment (amos-cha) [inline .env.development:13]: remember to revert your `.env` files

File: src/views/RecIM/components/List/Listing/index.jsx [hunk 2]
```diff
@@ -299,6 +304,12 @@ const ParticipantListing = ({
     const loadAttendanceCount = async () => {
       setAttendanceCount(await getParticipantAttendanceCountForTeam(teamID, participant.Username));
     };
+
+    const loadStatusType = async () => {
+      setStatusTypes(await getParticipantStatusTypes());
+    };
+    loadStatusType();
+    //loadUserInfo();
```
Comment (amos-cha) [inline src/views/RecIM/components/List/Listing/index.jsx:312]:
```suggestion
```

**Review by amos-cha — CHANGES_REQUESTED** [review]
Good work so far, it would be beneficial for us if you also uploaded a screenshot of what the change looks like! You can do this with a `ctrl + shift + s` and just paste in the text for github!

**Review by amos-cha — CHANGES_REQUESTED** [review]

**Pull request closed without merging**